data/transactions/logic/assembler.go

```diff
@@ -980,6 +1034,27 @@ func assembleGtxnsa(ops *OpStream, spec *OpSpec, args []string) error {
 	return nil
 }
 
+func assembleGtxnsas(ops *OpStream, spec *OpSpec, args []string) error {
+	if len(args) != 1 {
+		return ops.error("gtxnsas expects one immediate argument")
```

Comment:
Let's try to use %s and spec.Name as often as possible.

Reply:
Agreed -  I think we can potentially save ourselves time in the future by doing this.